testing/py-virtualenvwrapper/APKBUILD

```diff
@@ -0,0 +1,50 @@
+# Contributor: Herve Beraud <herveberaud.pro@gmail.com>
+# Maintainer: Herve Beraud <herveberaud.pro@gmail.com>
+pkgname=py-virtualenvwrapper
+pkgver=4.8.0
+pkgrel=0
+pkgdesc="Enhancements to virtualenv"
+url="https://bitbucket.org/virtualenvwrapper/virtualenvwrapper/"
+arch="noarch"
+license="MIT"
+depends="python"
+makedepends='python2-dev python3-dev py-setuptools py-pbr'
+install=""
+subpackages=""
```

Comment:
You have to declare subpackages. It’s all at the already mentioned wiki page: https://wiki.alpinelinux.org/wiki/APKBUILD_examples:Python.